Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMultiView.css

```diff
@@ -41,7 +41,8 @@ div.tabbedMultiViewActiveView > div.contentBorder
 {
   padding: var(--tabbedmultiview-view-padding);
   overflow: auto;
-  border: 1px solid transparent;
+  /* Offset for focus-border */
+  border: var(--remotion-themed-border-placeholder);
```

Comment:
we should never use literal values if there is a correspnding variable. And since we keep needing the placeholder border, I introduced it.